tests/baselines/reference/nodeModules1(module=node16).errors.txt

```diff
@@ -15,27 +14,16 @@ index.cts(52,22): error TS1471: Module './index' cannot be imported using this c
 index.cts(59,22): error TS1471: Module './subfolder2/another' cannot be imported using this construct. The specifier only resolves to an ES module, which cannot be imported with 'require'. Use an ECMAScript import instead.
 index.cts(60,22): error TS1471: Module './subfolder2/another/' cannot be imported using this construct. The specifier only resolves to an ES module, which cannot be imported with 'require'. Use an ECMAScript import instead.
 index.cts(61,22): error TS1471: Module './subfolder2/another/index' cannot be imported using this construct. The specifier only resolves to an ES module, which cannot be imported with 'require'. Use an ECMAScript import instead.
-index.cts(75,14): error TS2712: A dynamic import call in ES5/ES3 requires the 'Promise' constructor.  Make sure you have a declaration for the 'Promise' constructor or include 'ES2015' in your '--lib' option.

```

Comment:
It seems strange for this to be changed, but the testing harness eagerly resolved the target such that the "implicit" target for node16/nodenext was not used! Now it is.